docs/lib/storage/fragments/flutter/upload/upload-create-file.md

```diff
@@ -0,0 +1,26 @@
+Here we use the [path_provider](https://pub.dev/packages/path_provider) package to get a directory where we can create a temporary file for uploading.
```

Comment:
Nit: Probably this can be made a common fragment since its used in multiple places.
Can be in a followup PR

Reply:
Yep, agreed.